cluster/images/etcd/Makefile

```diff
@@ -62,7 +62,7 @@ endif
 # This option is for running docker manifest command
 export DOCKER_CLI_EXPERIMENTAL := enabled
 # golang version should match the golang version from https://github.com/coreos/etcd/releases for the current ETCD_VERSION.
-GOLANG_VERSION?=1.12.9
+GOLANG_VERSION?=1.12.12
```

Comment:
nit: can we work with etcd folks to bump go version?

Reply:
Do you mean to bump go version for etcd v3.4 release branch or etcd master branch (for v3.5)?

Reply:
I meant the latter. The former would also be great, but it's less critical.

Reply:
etcd master branch is currently built with go 1.13.3. I am using go 1.12.12 in this PR because etcd v3.4.4 is built with that go version.

Reply:
OK great - that sounds totally fine.